02-ml-normalization.mm1

```diff
@@ -1,6 +1,11 @@
 import "00-matching-logic.mm0";
 import "01-propositional.mm1";
 
+--- TODO: Remove me. We should instead be adding a positivity hypothesis
+--- to each theorem that needs this.
+theorem sSubstitution_in_mu_disjoint {X: SVar} (psi: Pattern X) {Y: SVar} (phi: Pattern X Y):
+  $ Norm (s[ psi / X ] mu Y phi) (mu Y (s[ psi / X ] phi)) $
+= '(sSubstitution_in_mu sFresh_disjoint);
```

Comment:
Could you please be more specific when you need more information please.

If you're referring to the comment then: We've got a lot theorems that assume disjointedness, when freshness would suffice. Going from the fresh version to the disjoint version is as easy as adding a single argument, `sFresh_disjoint` to the theorem's application. On the other hand, going the other way around involves going through each of the uses of `_disjoint` theorems and proving the more general version. Whole hierarchies of theorems may need to be generalized. I feel this is something we should try to avoid.

Reply:
If you like I can create an issue or add the above comment to the TODO comment.

Reply:
Ah, looks like this isn't actually used as I thought it was. I've removed it.